studio/libs/maslib.js

```diff
@@ -1,6 +1,6 @@
 const getMiloLibs = () => {
     const milolibs = new URLSearchParams(window.location.search)?.get('milolibs');
-    if (!milolibs) return 'https://www.adobe.com';
+    if (!milolibs) return 'https://ccd--milo--adobecom.hlx.live';
```

Comment:
can we rather add localeStorage for milolibs? and we leave real domain in code?

Reply:
When visitors come to https://main--mas--adobecom.hlx.live/studio.html, I would like it to work with the latest fixes without having to add `milolibs` params.
https://www.adobe.com is lagging behind and will be the same for sometime.
At someone we can revert this change, but I think it is too early for www.adobe

Reply:
>can we rather add localeStorage for milolibs?
I would keep request parameter, it's so much easier to use.
you can share a link with someone or open 2 tabs with different branches

Reply:
and yes, it doesn't look like a good idea to me to point to a feature branch.
if you want to share the middle state with a visitor just add milolibs, but we should bound studio to main

Reply:
i would keep query param, just add local storage for devs like us tired of adding the parameter